dockerfile/security/missing-user.yaml

```diff
@@ -4,6 +4,7 @@ rules:
     - pattern: |
         CMD $...VARS
     - pattern-not-inside: |
+        ...
         USER $USER
         ...
```

Comment:
To avoid a starting ellipses, maybe we could do something like this?

```
    - pattern: |
        CMD $...VARS
    - pattern-not-inside:
        USER $USER
        ...
        CMD $...VARS
    - pattern-not-inside:
        CMD $...VARS
        ...
        USER $USER
```